tinygrad/renderer/assembly.py

```diff
@@ -64,6 +64,10 @@ def gate_rep(root, x, y, z, k):
     root.vin = (x,y,z,new)
     return ld_rep(root,x,y)
 
+  def mulacc(root, muls, non_muls):
+      root.arg = TernaryOps.MULACC
```

Comment:
Wrong spacing

Reply:
updated, not sure how to make this a lambda now that dtype check is back

Reply:
Check the dtype in the pattern matcher. I'd be fine with extending it to support a set of dtypes.

Reply:
> Check the dtype in the pattern matcher. I'd be fine with extending it to support a set of dtypes.

Updated